### PR TITLE
Feat/styled component

### PR DESCRIPTION
**Pull request by wincentrtz**
This is what i have done:
- Installing styled component dependencies
- Doing some folder refactoring on hello component
- Adding styled component on hello component

### CODE REVIEW

File: src/client/components/hello/style.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components'
+
+export const Container = styled.div``
```
Comment (anikethsaha) [inline src/client/components/hello/style.js]: Can you add little bit of styles here 
Like center , adding fonts and making the `hello` little bit bigger !
Else everything looks good !

File: src/client/router/appSwitch.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
-import Hello from '../components/hello.jsx';
+import Hello from '../components/hello';
 
 const AppSwitch = () => (
     <Switch>
-        <Route exact path='/' component={ Hello } exact />
```
Comment (anikethsaha) [inline src/client/router/appSwitch.jsx:7]: Thanks for pointing this

**Review by anikethsaha — CHANGES_REQUESTED** [review]
Looks good to me 
Left few suggestions
also, what do you think of making `hello.js` -> `hello.jsx` ?
You can convert that to class component too if you want!

**Comment by anikethsaha** [comment]
You can now make changes in your local machine and then push it to your branch itself. Thos will be reflected here automatically

**Comment by wincentrtz** [comment]
Actually there's no hello.js in this commit 

I move it into a hello folder and put index.js and style.js inside. I guess that's better and it will make our import statement shorter too.

But if we want to keep it as hello.jsx, we can create a folder called styled with same level with __test__,

This is the current structured: 
image


This is the structured I suggest if we want to keep file name as hello.jsx: 
image

What do you think @anikethsaha ?

**Comment by anikethsaha** [comment]
Ohh sorry, my bad...I meant `index.js` -> `index.jsx` ?
No, you can keep it `index` itself ...the folder name `hello` looks good....we can keep everything related to that in that folder like tests and all 👍

**Comment by wincentrtz** [comment]
If we change it to index.jsx i guess there'll be a problem for javascript import statement. Where they can't detect index.jsx itself.

This is our previous import statement:
image
We dont need to specify index.js at the end of import statement, because it's index.js

but when we're using jsx i'm facing this kind of issue
image
and we need to add index.jsx at the end of import statement
is it okay?

**Comment by anikethsaha** [comment]
Yea I faced that too. Actually I changed all components to `jsx` and added `.jsx` at the end of the imports. I don't know why it's like that maybe some error in placing babel plugin.
Ok, you can leave it as it is. No need to change it for now.
Add some styles to the div (container) https://github.com/anikethsaha/MERN/pull/114#discussion_r334224775
and then we are good to go for merging it 👍

**Comment by wincentrtz** [comment]
Awesome then!
I have pushed the new one @anikethsaha 

Please check it once more :)

Thank you

**Review by anikethsaha — APPROVED** [review]
Good work 🎉

**Comment by anikethsaha** [comment]
@all-contributors please add @wincentrtz for code

**Pull request merged**